backend/services/theme_service.py

```
@@ -13,7 +13,8 @@
 from fastapi import HTTPException, Depends
 import asyncpg
 import json
-from pathlib import Path # CORE.MD: Import 'pathlib' for robust path resolution.
+from pathlib import Path 
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)